alveo-hbm/Gen3x4_Guide.md

```diff
@@ -0,0 +1,52 @@
+# DPUv3E on U50 with Gen3x4 Platform Usage Guide
+DPUv3E on U50 with Gen3x4 Platform is in Early-Access stage now, you could follow the instruction below to try it. It will bring 14% higher performance than the default Gen3x16 Platform version. 
+### Redhat/Centos 7.4-7.6 is supported for this version in Early-Access stage.
+
+## 1. Card preparation
+Firstly please ensure latest version XRT has been installed correctly. Then use following command to check the version of Depolyment shell installed on host and card:
+~~~
+/opt/xilinx/xrt/bin/xbutil scan
+~~~
+You could see the running log similar to following:
+~~~
+INFO: Found total 1 card(s), 1 are usable
+-------------------------------------------------------------------------------
+System Configuration
+OS name:        Linux
+Release:        4.15.0-91-generic
+Version:        #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020
+Machine:        x86_64
+Model:          PowerEdge R740xd
+CPU cores:      96
+Memory:         385419 MB
+Glibc:          2.27
+Distribution:   Ubuntu 18.04.1 LTS
+Now:            Tue May  5 22:42:01 2020
+-------------------------------------------------------------------------------
+XRT Information
+Version:        2.5.309
+Git Hash:       9a03790c11f066a5597b133db737cf4683ad84c8
+Git Branch:     2019.2_PU2
+Build Date:     2020-02-23 18:52:05
+XOCL:           2.5.309,9a03790c11f066a5597b133db737cf4683ad84c8
+XCLMGMT:        2.5.309,9a03790c11f066a5597b133db737cf4683ad84c8
+-------------------------------------------------------------------------------
+ [0] 0000:af:00.1 xilinx_u50_gen3x4_xdma_201920_3 user(inst=129)
+~~~
+
+If the last line of platform information is not **xilinx_u50_gen3x4_xdma_201920_3**, please run script **card_setup.sh** from host to finish the installation of gen3x4 platform and card flashing. You will need sudo privilege to finish this step.
+~~~
+./card_setup.sh
```

Comment:
this instruction should be preceded by "sudo" right?

Reply:
Necessary 'sudo' is included in the command line of the scripts.